x-pack/plugins/maps/public/shared/layers/vector_layer.js

```diff
@@ -569,8 +537,8 @@ export class VectorLayer extends AbstractLayer {
     return this.isVisible() && this._source.canFormatFeatureProperties();
   }
 
-  getFeatureByFeatureById(id) {
-    const featureCollection = this._getSourceFeatureCollection(id);
+  getFeatureById(id) {
```

Comment:
good catch on typo.

not sure how we should make the distinction, in terminology between the two-types of eature-ids.
- feature.id as it is coming from the geojson (presumable, this identifies an object in the "real world"
- dynamically generated featureId, created by the maps-app, on a per-request basis to uniquely identify a feature in the response (needed for tooltip-purposes).

(if we need to make this distinction at all...)